### PR TITLE
add json field; indent

### PR DESCRIPTION
**Pull request by djspoons**

### CODE REVIEW

File: collector.proto [hunk 1]
```diff
@@ -12,89 +12,90 @@ option optimize_for = CODE_SIZE;
...
-	double double_value = 4;
-	bool bool_value = 5;
-    }
+  string key = 1;
+  oneof value {
+    string string_value = 2;
+    int64 int_value = 3;
+    double double_value = 4;
+    bool bool_value = 5;
+    string json_value = 6;
```
Comment (djspoons) [inline collector.proto]: This additional line is the only non-whitespace change.

**Comment by djspoons** [comment]
@bensigelman (and I think @bcronin) we've discussed this as a way for clients to indicate that they are sending JSON.

File: collector.proto (hunk 1 above)
Comment (bhs) [reply collector.proto]: re the whitespace -- I kinda like 4-space tabstops, but whatever...

File: collector.proto [hunk 2]
```diff
@@ -12,89 +12,90 @@ option optimize_for = CODE_SIZE;
...
-    string key = 1;
-    oneof value {
-	string string_value = 2;
-	int64 int_value = 3;
-	double double_value = 4;
-	bool bool_value = 5;
-    }
+  string key = 1;
+  oneof value {
+    string string_value = 2;
```
Comment (bhs) [inline collector.proto]: let's add a comment explaining the distinction with `json_value`

File: collector.proto (hunk 1 above)
Comment (bhs) [reply collector.proto]: let's add a comment explaining that the `json_value` MUST be well-formed JSON (i.e., no hard string truncations allowed).

**Review by bhs — APPROVED** [review]
LGTM

**Pull request merged**